trino/overlays/prod/aws-secret.enc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: prod-aws-secret
+stringData:
+    AWS_ACCESS_KEY_ID: ENC[AES256_GCM,data:Zgsw//wktmGxJ5m9sH7LcJkjyFY=,iv:CzHAYMeuu2iXbiw2SEER1K/7L9b4CConAmBnL4KFnDk=,tag:O0UG/3CwNrRijfd/nWbclg==,type:str]
+    AWS_SECRET_ACCESS_KEY: ENC[AES256_GCM,data:MI5pFS7My1q3vknHG6Dt1FOjK6TKTZhqAU1jkh1h04Qs5JdNXzBT1w==,iv:38CUSop38oTHCo7p267vSsxDJ9b2bk6/EfldmyVBo2s=,tag:MpZru7Ffj0cl1zlBUazv+g==,type:str]
+    BUCKET: ENC[AES256_GCM,data:K7e3rORCSEjwg8epmOoKomYNinizQv1/hx3RoIm3,iv:LLZ9m65W5YtGgSw0cPKJJ+ILNPlWV895C5YB+GSNwU8=,tag:MSp0d6TXFj2FBWGFj4/IIg==,type:str]
```

Comment:
We should use the playpen bucket and credentials here, same as in stage and dev.